tests/functional/spec/interstitial_definition.spec.js

```diff
@@ -8,36 +8,20 @@ describe("Component: Interstitial Definition", () => {
 
     it("When there is a definition on an interstitial, then the page is displayed correctly", () => {
       expect($(InterstitialDefinitionPage.definitionTitle(1)).isDisplayed()).to.be.true;
```

Comment:
Similarly to your other changes, this assertion can be removed since it is always `true`, always displayed now (and only checking the contents has any value)?